app/src/main/java/nerd/tuxmobil/fahrplan/congress/schedule/FahrplanFragment.java

```diff
@@ -98,7 +101,7 @@
 
     private int mDay = 1;
 
-    public static Context context = null;
+    public Context context = null;
```

Comment:
IntelliJ linter warned me "StaticFieldLeak" rule. Did not find any official documentation. The next best match is https://blog.mindorks.com/understanding-context-in-android-application-330913e32514. I think it is a preventive measure.

Reply:
Thank you for the link. I keep this in mind. Let's have a look at this topic in the future.

Reply:
As I understood, this change is not directly related to the commits of this branch. Therefore, please extract this change into a separate commit and state the motivation for the change. I like to avoid mixing it into the other commits.